code/game/objects/items/storage/toolbox.dm

```diff
@@ -19,7 +19,7 @@
 	pickup_sound =  'sound/items/handling/toolbox_pickup.ogg'
 	material_flags = MATERIAL_COLOR
 	var/latches = "single_latch"
-	var/has_latches = TRUE
+	var/has_latches = FALSE
```

Comment:
if you are going to remove the latches, please remove the code associated with them